docs/source/deployment/databricks.md

```diff
@@ -60,22 +60,20 @@ You should get a similar output:
 ```console
 ...
 [08/09/22 11:23:30] INFO     Model has accuracy of 0.933 on test data.                                        nodes.py:74
-                    INFO     Saving data to 'metrics' (MetricsDataSet)...                             data_catalog.py:382
                     INFO     Completed 3 out of 3 tasks                                           sequential_runner.py:85
                     INFO     Pipeline execution completed successfully.                                      runner.py:89
 ```
 ### 3. Create a Databricks cluster
 
-If you already have an active cluster with runtime version `7.1`, you can skip this step. Here is [how to find clusters in your Databricks workspace](https://docs.databricks.com/clusters/clusters-manage.html).
+If you already have an active cluster with runtime version `7.3`, you can skip this step. Here is [how to find clusters in your Databricks workspace](https://docs.databricks.com/clusters/clusters-manage.html).
 
 Follow the [Databricks official guide to create a new cluster](https://docs.databricks.com/clusters/create-cluster.html). For the purpose of this tutorial (and to minimise costs) we recommend the following settings:
-* Runtime: `7.1 (Scala 2.12, Spark 3.0.0)`
+* Runtime: `7.3 (Scala 2.12, Spark 3.0.1)`
```

Comment:
✅ I think this is the best version to specify for now, though we may have to revisit this when we rewrite as part of https://github.com/kedro-org/kedro/issues/2185.